### PR TITLE
BETA: Register ohxxm.is-a.dev

### PR DESCRIPTION
**Pull request by ohxxm**
Added `ohxxm.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by ohxxm — COMMENTED** [review]
fix ohxxm.json file

File: domains/ohxxm.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "email": "ohmrc2545@gmail.com"
+    },
+    "record": {
+        "CNAME": "ohxxm.github.io"
+        "A": [
+            "185.199.108.153",
+            "185.199.109.153",
+            "185.199.110.153",
+            "185.199.111.153"
+        ],
```
Comment (CuteDog5695) [inline domains/ohxxm.json:13]: Cant use CNAME with any other record

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/ohxxm.json (hunk 1 above)
Comment (ohxxm) [reply domains/ohxxm.json:13]: mb im sped when it comes to cname and records.

**Review by ohxxm — COMMENTED** [review]
fixes issue

File: domains/ohxxm.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "owner": {
+        "username": "ohxxm",
+        "email": "ohmrc2545@gmail.com"
+    },
+    "record": {
+        "CNAME": "ohxxm.github.io"
```
Comment (CuteDog5695) [inline domains/ohxxm.json:7]: cant use CNAME with any other record

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**